platform/ext/target/nrf5340/target_cfg.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2017-2019 Arm Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __TARGET_CFG_H__
+#define __TARGET_CFG_H__
+
+#include "platform/ext/common/uart_stdout.h"
+#include "tfm_plat_defs.h"
```

Comment:
`<tfm_plat_defs.h>` would be more appropriate.

Reply:
I copied how it is done for other platform-specific headers

Reply:
`""` means: first look in the same directory, then in include paths, `<>` means start from include paths That's why I thought the latter would be more appropriate. But that's just a detail, I won't insist on it.